app/helpers/date_helpers.py

```diff
@@ -0,0 +1,30 @@
+from isoweek import Week
+from datetime import date
+import re
+from typing import Optional
+
+def date_to_iso_week(date_str: str) -> Optional[Week]:
```

Comment:
Why not strptime ?
or python 3.7's fromisoformat(date_string)
or even https://pypi.org/project/python-dateutil/

Reply:
I was gonna use py3.7, but I couldn't guarantee I'll have it on my computer. That being said, I'm not using this file so I can remove the entire thing

Reply:
_oh_ okay...